understanding/consistent-help.html

```diff
@@ -53,322 +52,225 @@
+                  <h3>ヘルプメカニズム</h3>
                   
                   
-                  <p>Typical help mechanisms include:</p>
+                  <p>典型的なヘルプメカニズムには、次のものがある:</p>
                   
                   <ul>
                      
-                     <li>Human contact details such as a phone number, email address, hours of operation.</li>
+                     <li>電話番号、電子メールアドレス、営業時間などの人間への連絡先の詳細。</li>
                      
-                     <li>Human contact mechanism such as a messaging system, chat client, contact form, social
-                        media channel.
+                     <li>メッセージングシステム、チャットクライアント、問い合わせフォーム、ソーシャルメディアチャネルなどの人間への連絡メカニズム。
                      </li>
                      
-                     <li>Self-help option such as an up-to-date Frequently Asked Questions, How Do I page,
-                        Support page.
+                     <li>最新のよくある質問、操作方法ページ、サポートページなどの自己解決のためのオプション。
                      </li>
                      
-                     <li>A fully automated contact mechanism such as a chatbot.</li>
+                     <li>チャットボットなどの完全に自動化された連絡メカニズム。</li>
                      
                   </ul>
                   
                   
-                  <p>The order of the types of help listed in the Success Criterion does not imply priority.</p>
+                  <p>達成基準に記載されているヘルプの種類の順序は、優先順位を意味するものではない。</p>
                   
                </section>
                <section id="support-for-coga">
                   
-                  <h3>Support for people with cognitive and learning disabilities</h3>
+                  <h3>認知障害及び学習障害のある人々へのサポート</h3>
                   
                   
-                  <p>This section is not required by the Consistent Help success criterion, but provides
-                     advice related to <a href="https://www.w3.org/TR/coga-usable/#support-user-story">Making Content Usable for People with Cognitive and Learning Disabilities</a>.
+                  <p>このセクションは、一貫したヘルプの達成基準では必須ではないが、<a href="https://www.w3.org/TR/coga-usable/#support-user-story">Making Content Usable for People with Cognitive and Learning Disabilities</a> に関連するアドバイスを提供する。.
                   </p>
                   
                   
-                  <p>The human contact details enable users to connect with the  organization or the part
-                     of the organization that can assist with the content. For example, an online jobs
-                     / recruitment portal may provide a contact method for the team that supports the recruitment
-                     portal and not a catch-all for the entire company. Each layer of contact added prolongs
-                     the time before the user will receive help.
+                  <p>人間への連絡先により、利用者がコンテンツを支援できる組織又は組織の一部と連絡することを可能にする。例えば、オンラインの求人／採用ポータルは、会社全体の連絡方法ではなく、採用ポータルをサポートするチームへの連絡方法を提供する場合がある。連絡先の層が追加されるたびに、利用者が助けを受けるまでの時間が長くなる。
```

Comment:
2つめと3つめのセンテンスのつながりがスムーズになるように、若干表現を変えてみました。いかがでしょうか。

```suggestion
                  <p>人間への連絡先により、利用者がコンテンツを支援できる組織又は組織の一部と連絡することを可能にする。例えば、オンラインの求人／採用ポータルは、会社全体の連絡方法ではなく、採用ポータルをサポートするチームへの連絡方法を提供する場合がある。連絡先の層が追加されると、利用者が助けを受けるまでの時間が長くなるからである。
```

Reply:
むう…意訳した方がよいのかなんてもなので、主査レビュー時に判断いただきましょう（雑

Reply:
part ofには「～の一員」いうのがあるので、担当者とするほうがよいかもしれません。いただいたコメントを踏まえてこれはどうでしょう…？

```suggestion
                  <p>人間への連絡先によって、利用者は、組織又は組織の担当者と連絡をとり、コンテンツについての支援を受けられる。例えば、オンラインの求人／採用ポータルは、会社全体の包括的な連絡方法ではなく、その採用ポータルをサポートするチームへの連絡方法を提供する場合がある。連絡先の階層が増えるほど、利用者が助けを受けるまでの時間が長くなる。
```

Reply:
組織又は組織の担当者
→
組織又は組織の担当部署